src/set_theory/cardinal/cofinality.lean

```diff
@@ -705,13 +705,25 @@ local infixr ^ := @pow cardinal.{u} cardinal cardinal.has_pow
 def is_limit (c : cardinal) : Prop :=
 c ≠ 0 ∧ ∀ x < c, succ x < c
 
+theorem is_limit.ne_zero {c} (h : is_limit c) : c ≠ 0 :=
+h.1
+
+theorem is_limit.succ_lt {x c} (h : is_limit c) : x < c → succ x < c :=
+h.2 x
+
 /-- A cardinal is a strong limit if it is not zero and it is
   closed under powersets. Note that `ω` is a strong limit by this definition. -/
 def is_strong_limit (c : cardinal) : Prop :=
 c ≠ 0 ∧ ∀ x < c, 2 ^ x < c
 
+theorem is_strong_limit.ne_zero {c} (h : is_strong_limit c) : c ≠ 0 :=
+h.1
+
+theorem is_strong_limit.power_lt {x c} (h : is_strong_limit c) : x < c → 2 ^ x < c :=
```

Comment:
```suggestion
theorem is_strong_limit.two_pow_lt {x c} (h : is_strong_limit c) : x < c → 2 ^ x < c :=
```

Reply:
We use `power` to distinguish cardinal exponentiation from that of monoids. I'll gladly add `two` to the lemma name though.